drivers/spi/spi_context.h

```diff
@@ -292,6 +302,13 @@ static inline void spi_context_complete(struct spi_context *ctx,
 #endif /* CONFIG_SPI_ASYNC */
 }
 
+#ifdef DT_SPI_CTX_HAS_NO_CS_GPIOS
+#define spi_context_cs_configure_all(...)
+#define spi_context_cs_get_all(...)
+#define spi_context_cs_put_all(...)
```

Comment:
These functions have `int` return types. Could be defined as `0`, not sure if this could cause other issues?